### ISSUE TITLE
Already trained model wants to add new videos to train on top of the original one

### ISSUE THREAD

**Issue by lBWtt** [DeepLabCut/DeepLabCut#1998]
**Is your feature request related to a problem? Please describe.**

I have trained a snapshot of 20,000 times, I now want to add some new videos and mark them to train again on the basis of 20,000 times. In order to increase the accuracy of my model(Not starting over, but building on what we had)

**Comment by jeylau** [comment]
You should use `deeplabcut.add_new_videos(config, videos, extract_frames=True)`. Then annotate the newly extracted frames, create a new training dataset with `deeplabcut.create_training_dataset`, and fine-tune your network making sure that you edit `init_weights` in the dlc-models/train/pose_cfg.yaml to point to your 20k snapshot prior to retraining.

**Comment by lBWtt** [comment]
> 应使用 。然后对新提取的帧进行注释，使用 创建新的训练数据集，并微调网络，确保在重新训练之前在 dlc 模型/train/pose_cfg.yaml 中进行编辑以指向 20k 快照。`deeplabcut.add_new_videos(config, videos, extract_frames=True)``deeplabcut.create_training_dataset``init_weights`

Thank you ！

**Comment by lBWtt** [comment]
> You should use `deeplabcut.add_new_videos(config, videos, extract_frames=True)`. Then annotate the newly extracted frames, create a new training dataset with `deeplabcut.create_training_dataset`, and fine-tune your network making sure that you edit `init_weights` in the dlc-models/train/pose_cfg.yaml to point to your 20k snapshot prior to retraining.

Can I reset init_weights in CoLab and which suffix to use?

**Comment by MMathisLab** [comment]
yes; you can open the `pose_cfg` within colab (left side folders)and set the int_weight; if you right click on the file, you can get the full path prefix :)

**Comment by lBWtt** [comment]
> yes; you can open the `pose_cfg` within colab (left side folders)and set the int_weight; if you right click on the file, you can get the full path prefix :)

Thank you for your reply. I ran the reload model on my local computer. Do you know what happened when I encountered the following error? 
The path is right.


![J$G{W7SYC4$B4LQW%M6Q} N](https://user-images.githubusercontent.com/105534764/195132038-5a1f53c5-2e58-4c90-a146-09f8a6d55266.png)

![S% %3XN37~S92Z3VCU G$NX](https://user-images.githubusercontent.com/105534764/195132145-34423107-7ec2-49b8-bc0f-3aff6fa1bf0c.png)

![0HTZ%($}N1FVT1G)BF%6FTI](https://user-images.githubusercontent.com/105534764/195132193-1d242b7d-0ee9-4499-8f17-d1af63485b1a.png)

![image](https://user-images.githubusercontent.com/105534764/195132282-e5d2f2e5-e416-4c0b-ade7-4d2cb3242979.png)

**Comment by MMathisLab** [comment]
haven't seen that 👀
